### PR TITLE
Docs | Update Community.md and contributing/translation  (#31883)

### LINKED ISSUE #31883: community.md - translations as a form of contributing are redundant
## Summary

Hi! Recently I wanted to contribute to [gatsby/gatsby-pl](https://github.com/gatsbyjs/gatsby-pl), but after a short conversation I've found out that translations are on hold (See [this comment](https://github.com/gatsbyjs/gatsby-pl/issues/1#issuecomment-859770467)), but there's no information about it on gatsbyjs.com and it still encourages to contribute in translations.

Could we please update [community.md](https://github.com/weronikadominiak/gatsby/blob/master/docs/contributing/community.md#ways-to-contribute) (and potentially [contributing/translation](https://github.com/weronikadominiak/gatsby/tree/master/docs/contributing/translation))?

I'm happy to create PR with the change if you would confirm the information above that translations are no longer required.

### Motivation

Someone might want to work on a translation that won't be used. 

## Steps to resolve this issue

Include an annotation that at the moment translations are on hold or remove them completely from the docs.

### PR DESCRIPTION
**Pull request by weronikadominiak**
## Description

Based on @meganesu [suggestion](https://github.com/gatsbyjs/gatsby/issues/31883#issuecomment-875040762) I used discord message to add information on translation pages, that translations are on hold.

## Related Issues

 Fixes #31883

### CODE REVIEW

File: docs/contributing/community.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The Gatsby community welcomes contributions. Please refer to the guides below on
 - [How to Open a Pull Request](/contributing/how-to-open-a-pull-request/)
 - [Make feature requests with an RFC](/blog/2018-04-06-introducing-gatsby-rfc-process/)
 - [Gatsby Style Guide](/contributing/gatsby-style-guide/)
-- [Gatsby Translations](/contributing/gatsby-docs-translation-guide/)
+- [Gatsby Translations - currently on hold](/contributing/translation/)
```
Comment (weronikadominiak) [inline docs/contributing/community.md:35]: I updated the link - it was redirecting to it.

File: docs/contributing/translation/index.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ title: Translating the Gatsby Documentation
 
 import LangList from "@components/lang-list"
 
+> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towwards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```
Comment (meganesu) [inline docs/contributing/translation/index.md:7]: Small typo fix

```suggestion
> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```

File: docs/contributing/translation/maintainers.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 title: Translation Maintainer Guide
 ---
 
+> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towwards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```
Comment (meganesu) [inline docs/contributing/translation/maintainers.md:5]: Same typo fix

```suggestion
> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```

File: docs/contributing/translation/new-translations.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
 title: Starting a New Translation
 ---
 
+> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towwards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```
Comment (meganesu) [inline docs/contributing/translation/new-translations.md:5]: Same typo fix

```suggestion
> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```

File: docs/contributing/translation/style-guide.md [hunk 5]
```diff
@@ -2,6 +2,8 @@
 title: Translation Style Guide
 ---
 
+> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towwards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```
Comment (meganesu) [inline docs/contributing/translation/style-guide.md:5]: Same typo fix

```suggestion
> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```

File: docs/contributing/translation/sync-guide.md [hunk 6]
```diff
@@ -2,6 +2,8 @@
 title: Keeping Translations Up-to-date
 ---
 
+> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towwards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```
Comment (meganesu) [inline docs/contributing/translation/sync-guide.md:5]: Same typo fix

```suggestion
> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```

File: docs/contributing/translation/translators.md [hunk 7]
```diff
@@ -2,6 +2,8 @@
 title: Contributing to a Translation
 ---
 
+> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towwards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```
Comment (meganesu) [inline docs/contributing/translation/translators.md:5]: Same typo fix

```suggestion
> ⚠️ Note: At the moment our localization efforts are on pause, as we shifted towards a [redesign of the docs](https://www.gatsbyjs.com/blog/announcing-new-gatsby-docs-site/). Right now, we're prioritizing a new version of the tutorial. As with all prioritization efforts, we will weigh this, amongst other potential features, fixes, and improvements, and may consider picking up on the internationalization efforts of 2020. The repositories inside the GitHub org will remain. For now we archived the respective language channels on our Discord and the role for language maintainers.
```

**Review by meganesu — COMMENTED** [review]
Thanks for making these changes! Added comments for one small typo fix, otherwise this looks good to me!

Thanks again for your help 💜 👏

File: docs/contributing/community.md (hunk 1 above)
Comment (meganesu) [reply docs/contributing/community.md:35]: Good catch! Thanks :)

**Review by meganesu — APPROVED** [review]

**Pull request merged**